src/components/Form/Step3/Step3.js

```diff
@@ -121,42 +108,34 @@ const Step3 = ({
   }
 
   return (
-    <>
-      <h2>Tell us about your ticket</h2>
-      {errorState.errors.length && errorState.continuePressed && (
-        <GenericError />
-      )}
+    <form onSubmit={handleSubmit} ref={formRef} autoComplete="on">
+      <SectionStepInfo
+        section="Section 2 of 3"
+        description="About your ticket"
+      />
+
+      {/* Show generic error message */}
+      {showGenericError}
 
       {/* This changes based on switch logic above */}
       {elementsToRender}
 
-      <LastUsed />
+      <LastUsed setCannotProcess={setCannotProcess} />
```

Comment:
When this errors like so:
![image](https://user-images.githubusercontent.com/13015201/102099930-c3d00e80-3e20-11eb-9de6-37f3b7a4c90f.png)

If I click the "call us" button and then click "back" I lose all my progress. maybe open in a new window? Or enable the back button to come back to this step?

Reply:
Confirmed as working